### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by djhi**
- [x] Don't polyfill features now supported in all major browsers
- [x] Upgrade tooling

### CODE REVIEW

**Comment by djhi** [comment]
I did not update the version as I don't know if you want this to be a major release

**Comment by djhi** [comment]
Weird, all tests pass locally

File: src/FakeRest.js [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
+import { Single } from 'Single';
 
-export default {
-    Server: Server,
-    FetchServer: FetchServer,
-    Collection: Collection,
-    Single: Single
-}
+export { Server, FetchServer, Collection, Single };
+export default { Server, FetchServer, Collection, Single };
```
Comment (fzaninotto) [inline src/FakeRest.js:7]: I don't understand why you need both export and export default

File: src/Server.js [hunk 2]
```diff
@@ -1,11 +1,8 @@
-import objectAssign from 'object.assign';
-import Collection from 'Collection';
-import Single from 'Single';
-import parseQueryString from 'parseQueryString';
+import { Collection } from 'Collection';
```
Comment (fzaninotto) [inline src/Server.js:1]: why don't you import from a path?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: src/Server.js (hunk 2 above)
Comment (djhi) [reply src/Server.js:1]: Weird, I'll fix it

File: src/FakeRest.js (hunk 1 above)
Comment (djhi) [reply src/FakeRest.js:7]: To not break backward compatibility by forcing users to import like this:
```
import * as FakeRest from 'fakerest';
```

This way, they can keep their old import:
```
import FakRest from 'fakerest';
```

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**